purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -74,6 +74,8 @@ open class PurchasesConfiguration(builder: Builder) {
          * Examples of this information include response times, cache hits or error codes.
          * This information will be anonymized so it can't be traced back to the end-user.
          * The default value is false.
+         *
+         * Diagnostics is only available in Android API 24+
```

Comment:
Do we need to put this in the docs too?

Reply:
We haven't added diagnostics to the docs yet since we were planning to add extra functionality before that, so nothing to update